### DISCUSSION TITLE
Badges for private commits

### DISCUSSION THREAD

**Discussion by Jakhongiirr** [community/community#55190] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, do only public commits get badges? I am qualified to earn some badges in private repos, but no badge. Thanks beforehand

**Comment by ademakdogan** [comment, marked as answer]
GitHub badges are generated based on the activity on your public repositories. If you have made contributions to private repositories that are not accessible to the public, then those contributions will not be considered for the badges. Only public contributions, such as commits, pull requests, issues, and other activities on public repositories count towards badges.

**Reply by Jakhongiirr** [reply]
clear, thank you!
